js_src/dart_env_dev.mjs

```diff
@@ -1,10 +1,3 @@
-if (typeof window.MemoryInfo == "undefined") {
-  if (typeof window.performance.memory != "undefined") {
-    window.MemoryInfo = function () {};
-    window.MemoryInfo.prototype = window.performance.memory.__proto__;
-  }
-}
-
```

Comment:
https://groups.google.com/a/dartlang.org/g/announce/c/YtDxtYwvuBA/m/y_oScJcVAgAJ?pli=1

its fixed in chrome 87 and dart 2.9.3 we shouldn't need this anymore right @greglittlefield-wf